thampi/lib/util.py

```diff
@@ -59,5 +59,17 @@ def parent_dir(relative_file, level=1):
     return os.path.abspath(os.path.join(os.path.dirname(relative_file), *parent_dots))
 
 
+def load_local_model(model_path, model_file):
+    import pickle
```

Comment:
Nice, python convention, though I forget it too sometimes, is to put imports on the top of the file.

Reply:
Ahh yup! I just followed the pattern used when loading the pickled model from S3. Updated